readme.md

```diff
@@ -53,12 +53,15 @@ Watch our [video](https://youtu.be/0-FugzVYJQU) to find out how to do this, or s
 
 To add a Bitbucket Server instance:
 
-1. In Jenkins go to **Jenkins** > **Manage Jenkins** > **Configure System**.
-2. Under **Bitbucket Server integration** click **Add a Bitbucket Server instance**.
-3. Enter instance details.
-4. Add or select a personal access token. 
-5. Add or select credentials. 
-6. Select **Save**.
+1. In Jenkins, go to **Jenkins** > **Manage Jenkins** > **Configure System**.
+2. Under **Bitbucket Server integration**, select **Add a Bitbucket Server instance** > **Instance details**.
+3. Enter these instance details:
+   - Instance name - Enter a name to help users identify this instance. 
+   - Instance URL - Enter the Bitbucket Server base URL. 
+   - Personal access token - Select a token from the list. Or to first add a token, select **Add**, select [Bitbucket personal access token](https://confluence.atlassian.com/display/BitbucketServer/personal+access+tokens) for **Kind**, and then enter a Token (with project admin permissions) and a Description. 
+   - Credentials (for build auth) - Select credentials from the list. Or to first add credentials, select **Add**, select **Jenkins**, select Username with password for **Kind**, and then complete the form. 
```

Comment:
@mhenschke-atl, Brent mentioned that users can use SSH credentials instead of username and password. Is this right? 

Note: if correct need to change intro above that refers to username and password creds.

Reply:
It is. However, global credentials have been deprecated entirely in https://github.com/jenkinsci/atlassian-bitbucket-server-integration-plugin/pull/200 so you can remove that line entirely.

Reply:
(I think it's safe to remove now given the feature doesn't actually work, so that doesn't need to wait for a release)

Reply:
Thanks @mhenschke-atl. 

It sounds like in this screenshot the credentials field has to go. Are any other changes needed to it? 

Alternatively, how can I see the latest UI? 

![image](https://user-images.githubusercontent.com/53456463/96401148-53d94980-121e-11eb-94f7-0e4141cdffd5.png)

Reply:
I was wondering about that. I will look into this.

Reply:
Hi @mhenschke-atl. Any update? I'm keen to close this out.

Reply:
Hi cam, I've checked and the credentials update has been merged to master- make sure you're on the latest checkout and you can see it.